Cargo.toml

```diff
@@ -65,8 +65,26 @@ serde_json = "1.0"
 thiserror = "1.0"
 
 [workspace.lints.clippy]
```

Comment:
i guess my question would be why are these lints not enabled by default? Do we want to enable so many? I usually follow the defaults for lints as there is sometimes a reason why they are not enabled yet. As otherwise this can become an explosion of debates.

Reply:
No clippy lints are enabled by default as far as I'm aware.